CONTRIBUTING.md

```diff
@@ -1,11 +1,54 @@
 # Contributing
 
-New proposals to the API spec should be submitted by opening a Github issue with a link to a Google Doc containing the proposal. Proposals should contain motivation, use cases, and detailed example of proposed changes to API interactions. Collaboration on the design, fleshing out of use cases, etc can occur as comment discussions in the Google Doc, as well as on our weekly calls.
+## Minor Changes
 
-Once the design has been finalized in the Google Doc, support for the change must be implemented in the service controller of a platform easily accessible to users. Currently Cloud Foundry is the most appropriate platform for this, but we look forward to other platforms adding support for the broker API soon, and one of those platforms could take the initiative of implementing support for the feature earlier. 
+Minor change proposals to the specification, changes such as editorial bugs
+or enhancements that do not modify the semantics of the specification or
+syntax of the API, can be suggsted via a Github Issue or Pull Request (PR).
 
-Once support for the change has been implemented, a PR should be to this repo for the change to the broker API specificaion. By this point, the API interactions should be well understood and there should be no technical surprises; we expect the only discussion necessary on PRs to be for wordsmithing and formatting.
+If there may be a need for some discussion around how best to
+address the concern then opening an Issue, prior to doing the work to develop
+a PR, would be best.  These minor issues do not need a formal review, per the
+[process](#prissue-review-process) described below, rather the issue
+should be used to come to a conscensus around a PR that will eventually be
+submitted.
 
-All contributors sending Pull Requests (PRs) must have a Contributor License Agreement 
-on file as either an [individual](https://www.cloudfoundry.org/pdfs/CFF_Individual_CLA.pdf) 
+If the proposed change is non-controversial (e.g. a typo) then a PR can be
+submitted directly without opening an issue.
+
+Either way, once a PR is submitted it will be reviewed per the
+[process](#prissue-review-process) described below.
+
+## Design Change Proposals
+
+New design proposals to the API spec should be submitted by opening a Github issue with a link to a Google Doc containing the proposal. Proposals should contain motivation, use cases, and detailed example of proposed changes to API interactions. Collaboration on the design, fleshing out of use cases, etc can occur as comment discussions in the Google Doc, as well as on our weekly calls.
```

Comment:
Suggesting to refine the feedback solicitation aspect of the design review.

``
For changes largely impacting the SB API actors (platforms, service authors, etc...), it is recommended to solicit feedback from these actors and leave enough time (say 2 weeks) for feedback to be provided, and for the potentially received objections/suggestions to be handled.
``